### ISSUE TITLE
SpamassRules should be a string for Message

### ISSUE THREAD

**Issue by jooola** [mailjet/mailjet-apiv3-go#62]
While working with the https://api.mailjet.com/v3/REST/message endpoint, I faced the following issue:

`Error decoding API response: json: cannot unmarshal string into Go struct field Message.Data.SpamassRules of type []resources.SpamAssassinRule`

It seems like the SpamAssassinRule should either be a string or an array/object, but the client doesn't seem to be consistent with the API.

Since the API is the source of truth I prefer not proposing a PR.

**Comment by tbruyelle** [comment]
Hey there, I'm facing the exact same issue calling the message endpoint. Is there any update ?

**Comment by tbruyelle** [comment]
A simple workaround before a fix is provided : 

```diff
-       var resp []resources.Message
+       var resp []struct {
+               resources.Message
+               SpamassRules string `json:",omitempty"`
+       }
	err := mailjetClient.Get(&mailjet.Request{
		Resource: "message",
		ID:       id,
	}, &resp)
```

**Comment by xab3r** [comment]
fixed #70
